### PR TITLE
okcoin: Remove exchange implementation

### PR DESCRIPTION
**Pull request by thrasher-**
# PR Description

Removes OKCoin since OKX has reabsorbed all its minions (OKCoin Singapore, OKCoin US) back into the mothership 👽  

## Type of change

- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## How has this been tested

- [x] go test ./... -race
- [x] golangci-lint run

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation and regenerated documentation via the documentation tool
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally and on Github Actions with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: exchanges/order/order_types.go [hunk 1]
```diff
@@ -89,7 +89,7 @@ type Submit struct {
 
 	// Hidden when enabled orders not displaying in order book.
 	Hidden bool
-	// TradeMode specifies the trading mode for margin and non-margin orders: see okcoin_wrapper.go
+	// TradeMode specifies the trading mode for margin and non-margin orders: see okx_wrapper.go
 	TradeMode string
```
Comment (shazbert) [inline exchanges/order/order_types.go:93]: Rm not being used.

**Review by shazbert — CHANGES_REQUESTED** [review]
One thing from me. 🔥

**Review by gloriousCode — APPROVED** [review]
# 🫡 
F

tACK

**Review by shazbert — APPROVED** [review]
chACK! cya m8

**Pull request merged**